EDITORIAL CLOSING: XMPP Newsletter July 2022 (#1154)
xsf/xmpp.org: content/blog/2022-08-05-newsletter.md

By the way, xmpp.org features [a new page about XMPP RFCs](https://xmpp.org/rfcs/).

### Proposed

The XEP development process starts by writing up an idea and submitting it to the XMPP Editor. Within two weeks, the Council decides whether to accept this proposal as an Experimental XEP.

-   [WebSocket S2S](https://xmpp.org/extensions/inbox/websocket-s2s.html)
    -   This specification defines a procedure to make s2s XMPP connections over WebSocket.
-   [XMPP over QUIC](https://xmpp.org/extensions/inbox/xmpp-over-quic.html)
    -   This specification defines a procedure to make both c2s and s2s XMPP connections over the QUIC protocol instead of TCP+TLS.   

### New

> 

Author: Add RFC 9266

Author: Mistaken info

    - [On-Boarding Experience with XSF (Converse)](https://xmpp.org/2022/06/on-boarding-experience-with-xsf-converse/)
    - [ConverseJS: An in-depth view into my GSoC'22 project!](https://xmpp.org/2022/07/conversejs-an-in-depth-view-into-my-gsoc22-project/)

### XSF fiscal hosting projects

The XSF offers [fiscal hosting](https://xmpp.org/community/fiscalhost/) for XMPP projects. Please apply via [Open Collective](https://opencollective.com/xmpp). For more information, see the [announcement blog post](https://xmpp.org/2021/09/the-xsf-as-a-fiscal-host/). Current projects:

- [Bifrost bridge: Offline messages exchange between Matrix and XMPP](https://opencollective.com/bifrost-mam)
- [Mellium Co-op](https://opencollective.com/mellium)

## Events

- [XMPP Office Hours](https://wiki.xmpp.org/web/XMPP_Office_Hours): available on our [YouTube channel](https://www.youtube.com/channel/UCf3Kq2ElJDFQhYDdjn18RuA)
- [Berlin XMPP Meetup (remote)](https://mov.im/?node/pubsub.movim.eu/berlin-xmpp-meetup): monthly meeting of XMPP enthusiasts in Berlin, every 2nd Wednesday of the month

## Articles

> In general, articles and thoughts on the European Digital Markets Act:

Reviewer: ```suggestion
There are some articles and thoughts on the European Digital Markets Act: 

- [ec.europa.eu](https://ec.europa.eu/info/strategy/priorities-2019-2024/europe-fit-digital-age/digital-markets-act-ensuring-fair-and-open-digital-markets_en)
- [ZDF about interoperatibility](https://www.zdf.de/nachrichten/politik/whatsapp-threema-interoperabilitaet-100.html) [DE]
- German [freie-messenger.de about digital markets act](https://www.freie-messenger.de/en/begriffe/interoperabilitaet/dma)
- German [freie-messenger.de about interoperatibility](https://www.freie-messenger.de/en/begriffe/interoperabilitaet/gedanken)
```

> - https://ec.europa.eu/info/strategy/priorities-2019-2024/europe-fit-digital-age/digital-markets-act-ensuring-fair-and-open-digital-markets_en

Reviewer 2: Maybe replace these with the titles/authors and link them? Having bare URLs in the newsletter feels weird.

Author: 3ac5bc9d17b43d559f3c0e8600f35f26ae81b487

- https://www.zdf.de/nachrichten/politik/whatsapp-threema-interoperabilitaet-100.html
- https://www.freie-messenger.de/en/begriffe/interoperabilitaet/dma
- https://www.freie-messenger.de/en/begriffe/interoperabilitaet/gedanken

The site "Chat standard (XMPP)" with lots of information is avilable in Englisch now at [freie-messenger.de](https://www.freie-messenger.de/en/sys_xmpp)

> Axel from eversten.net published an [XMPP Push Notification Test for iOS](https://eversten.net/en/blog/notification/) covering latest Monal, Siskin IM and Snikket versions on different 

Reviewer 2: Linkify `eversten.net`?

Author: 3ac5bc9d17b43d559f3c0e8600f35f26ae81b487

> Axel from eversten.net published an [XMPP Push Notification Test for iOS](https://eversten.net/en/blog/notification/) covering latest Monal, Siskin IM and Snikket versions on different 

Reviewer: ```suggestion
Axel from [eversten.net](https://eversten.net) published an [XMPP Push Notification Test for iOS](https://eversten.net/en/blog/notification/) covering latest Monal, Siskin IM and Snikket versions on different 
```

server implementations. A good 15 mins read for both iOS users and their multi-platforms contacts alike.

JMP now delivers transcribed voicemail in many common languages, not just English. The [voicemail system has also been updated](https://blog.jmp.chat/b/july-newsletter-2022) to source names from more modern XMPP profiles including vCard4 and PEP Nickname.

> MangooseIM 5.1 was released last month, as announced in the [June 2022 Newsletter](https://xmpp.org/2022/07/the-xmpp-newsletter-june-2022/), and that brought improvements to the Inbox features, too. You can now read more about the new Inbox settings and scalability improvements in this [article](https://www.erlang-solutions.com/blog/updates-to-the-mim-inbox-in-version-5-1/).

Reviewer 2: I'd split this into two sentences, it's a bit of a runon. "MoongoseIM was released as announced. In addition to the previously mentioned features, this brought improvements to the Inbox feature. …"

Author: 3ac5bc9d17b43d559f3c0e8600f35f26ae81b487

## Software news

### Clients and applications

> The Mellium Dev Communiqué for July 2022 has been published! Important updates this month include an [implementation of the new RFC 9266 and a new tool for finding documentation related issues in Go projects](https://opencollective.com/mellium/updates/dev-communique-for-july-2022). Besides, Mellium has moved to Codeberg! The repositories can now be found at [codeberg.org/mellium](https://codeberg.org/mellium/) and the announcement post can be found on [Lemmy](https://community.xmpp.net/post/34677) or [Open Collective](https://opencollective.com/mellium/updates/mellium-development-has-moved-to-codeberg).

Reviewer 2: The links appear to have gotten changed and don't make sense anymore, the checkdoc tool is here: https://mellium.im/checkdoc/

s/Besides,/In addition,/

[Conversations 2.10.8](https://conversations.im/), a Jabber/XMPP client for Android, has been released.

[Gajim 1.4.6](https://gajim.org/post/2022-07-07-gajim-1.4.6-released/) and [1.4.7](https://gajim.org/post/2022-07-24-gajim-1.4.7-released/) have been released. These releases bring performance improvements, better file previews, and fix some bugs with the status icon and notifications. Emoji short code detection has been improved as well.

> Monal [version 5.2.0](https://github.com/monal-im/Monal/releases/tag/Build_iOS_796) has been release by the Monal development team! This release highlights are the new (private) push servers in Europe and US, display of transferred files in app's documents directory and several fixes on its encryption (OMEMO), QR code scanning and onboarding. Besides, Monal has also a [new webpage](https://www.monal-im.org/).

Reviewer: ```suggestion
Monal [version 5.2.0](https://github.com/monal-im/Monal/releases/tag/Build_iOS_796) has been released by the Monal development team! This release’s highlights are the new (private) push servers in Europe and the US, display of transferred files in app's documents directory, and several fixes regarding encryption (OMEMO), QR code scanning, and onboarding. Besides, Monal also features a [new webpage](https://www.monal-im.org/).
```

The site "Chat standard (XMPP)" with lots of information is avilable in Englisch now at [freie-messenger.de](https://www.freie-messenger.de/en/sys_xmpp)

Axel from [eversten.net](https://eversten.net/) published an [XMPP Push Notification Test for iOS](https://eversten.net/en/blog/notification/) covering latest Monal, Siskin IM and Snikket versions on different 
server implementations. A good 15 mins read for both iOS users and their multi-platforms contacts alike.

JMP now delivers transcribed voicemail in many common languages, not just English. The [voicemail system has also been updated](https://blog.jmp.chat/b/july-newsletter-2022) to source names from more modern XMPP profiles including vCard4 and PEP Nickname.

MoongoseIM 5.1 was released as announced, as announced in the [June 2022 Newsletter](https://xmpp.org/2022/07/the-xmpp-newsletter-june-2022/). In addition to the previously mentioned features, this brought improvements to the Inbox feature. You can now read more about the new Inbox settings and scalability improvements in this [article](https://www.erlang-solutions.com/blog/updates-to-the-mim-inbox-in-version-5-1/).

## Software news

### Clients and applications

> The Mellium Dev Communiqué for July 2022 has been published! Important updates this month include an [implementation of the new RFC 9266 and a new tool for finding documentation related issues in Go projects](https://opencollective.com/mellium/updates/dev-communique-for-july-2022). In addition, Mellium has moved to Codeberg! The repositories can now be found at [codeberg.org/mellium](https://codeberg.org/mellium/) and the announcement post can be found on [Lemmy](https://community.xmpp.net/post/34677) or [Open Collective](https://opencollective.com/mellium/updates/mellium-development-has-moved-to-codeberg).

Reviewer 2: Ah, my mistake, when I suggested the text we were talking about earlier I didn't put the links in. This long link looks really confusing to me though. Maybe something like this, instead:

> [The Mellium Dev Communiqué for July 2022](https://opencollective.com/mellium/updates/dev-communique-for-july-2022) has been published! Important updates this month include an implementation of the new [RFC 9266](https://www.rfc-editor.org/info/rfc9266) and a [new tool](https://mellium.im/checkdoc/) for finding documentation related issues in Go projects

Thanks, and sorry for the confusion!
